services/parent/pom.xml

```diff
@@ -72,6 +72,30 @@
       <artifactId>commons-logging-jboss-logging</artifactId>
     </dependency>
 
+    <!-- native build -->
+    <dependency>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-api</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-core</artifactId>
+      <version>2.20.0</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-1.2-api</artifactId>
+      <version>2.20.0</version>
+    </dependency>
+    <!--
+        fix https://github.com/apache/camel-quarkus/issues/5006#issuecomment-1611483154
+        this newer version of the library does not set the -H:-RunReachabilityHandlersConcurrently
+    -->
+    <dependency>
+      <groupId>com.google.api</groupId>
+      <artifactId>gax-grpc</artifactId>
+      <version>2.33.0</version>
+    </dependency>
```

Comment:
see `adapters/parent/pom.xml`

Reply:
I removed all these dependencies and replaced it with native-image.properties as you suggested.